README.md

```diff
@@ -84,6 +84,18 @@ You can use Base64 parameters for passing _small_ files to downstream builds:
 build job: 'downstream', parameters: [base64File(name: 'file', base64: Base64.encoder.encodeToString('hello'.bytes)))]
 ```
 
+## Usage with REST API
+
+You can pass file parameters to the REST API:
+
+```js
+const file = fileInput.files[0]; // a File object
+const body = new FormData();
+body.append('ICON_FILE', file); // will come through to the job as the named file parameter 'ICON_FILE'
+const request = new Request(`${buildAPI}buildWithParameters`, { method: 'POST', body });
+fetch(request); // omitted, crumb header and other credentials
```

Comment:
You should not need a crumb header—use an API token.

Reply:
Agreed

Reply:
> Good to document, though the example presumes familiarity with what I presume is Node.js. 

Actually it's really just a working example from js running in browser. I think something similar would also work in recent versions of nodejs as `fetch` has only been recently added. I'll add the `curl` example as well, but I don't really have a good way to test it.

Reply:
Got it, thanks.